gourmet/gtk_extras/dialog_extras.py

```diff
@@ -883,6 +885,7 @@ def saveas_file (title,
     retval = sfd.run()
     if not retval:
         return None,None
+    retval, = retval
```

Comment:
This only unpacks a single value, which is fine for saving. Maybe we could have an `assert len(retval == 1)` or at least a comment what this does.

Reply:
I would avoid `assert`s as much as possible. I my view, `assert` is only needed if we cannot continue with the wrong parameter (which is asserted). I'm not sure that here we have such a case. But I'm open for arguments :smiley:

Reply:
I would go for `if len(f) == 1:...` and rename `f` to `filename`

Reply:
(well, here, it would be `if len(retval) == 1:...`)